vhost-user-backend/src/lib.rs

```diff
@@ -33,6 +33,9 @@ pub use self::vring::{
     VringMutex, VringRwLock, VringState, VringStateGuard, VringStateMutGuard, VringT,
 };
 
+#[cfg(all(feature = "postcopy", feature = "xen"))]
+compile_error!("Both `postcopy` and `xen` features can not be enabled at the same time.");
```

Comment:
Ditto (document in the README and maybe a little comment here).

Reply:
Same as above